assets/js/components/higherorder/withdata.js

```diff
@@ -265,7 +267,7 @@ const withData = (
 
 			// If we have an error, display the DataErrorComponent.
 			if ( error ) {
-				return ( 'string' !== typeof error ) ? error : getDataErrorComponent( moduleName, error, layoutOptions.inGrid, layoutOptions.fullWidth, layoutOptions.createGrid, data );
+				return ( 'string' !== typeof error ) ? error : getDataErrorComponent( module, errorObj, layoutOptions.inGrid, layoutOptions.fullWidth, layoutOptions.createGrid );
```

Comment:
Let's revert the changes in this file accordingly.

Reply:
Yep, updated.